### DISCUSSION TITLE
Adding flashing arrow above minions head at Nex

### DISCUSSION THREAD

**Discussion by Newman1230** [runelite/runelite#17395] [Ideas]
I haven't been able to find a plugin that does this but sometimes Nex will not shout the minions name when it's time to attack them ex: "Glacias Don't fail me!" . My suggestion when it's time to kill the next minion have a flashing arrow or some type of notification above the minions head like similar to when doing clues it shows arrow telling the spot to dig. Or a plugin that simple tells you what the current phase is and when it's time to attack minion.
Thank you

**Comment by cdfisher** [comment]
I believe either of those would be in violation of [Jagex's guidelines for third-party clients](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1), specifically:

> Indicates the time where a boss mechanic may start or end

and/or

> Adds additional visual or audio indicators of a boss mechanic except in cases where this is a manually triggered external helper.

**Reply by Newman1230** [reply]
Oh really I didn't think it would since I figured it's just a visual queue and just a secondary confirmation that nex has summoned her next minion and you need to attack it. But makes sense thank you for that.
